lib/travis/build/script/julia.rb

```diff
@@ -90,14 +90,14 @@ def script
             if config[:coveralls]
               sh.cmd 'julia --color=yes -e "if VERSION < v\"0.7.0-DEV.5183\"; cd(Pkg.dir(\"${JL_PKG}\")); else using Pkg; end; Pkg.add(\"Coverage\"); using Coverage; Coveralls.submit(process_folder())"'
             end
-            
+
           end
           sh.else do
             sh.if '-a .git/shallow' do
               sh.cmd 'git fetch --unshallow'
             end
             # build
-            sh.cmd 'julia --color=yes -e "VERSION >= v\"0.7.0-DEV.5183\" && using Pkg; Pkg.clone(pwd()); Pkg.build(\"${JL_PKG}\")"', assert: true
+            sh.cmd 'julia --color=yes -e "if VERSION >= v\"0.7.0-DEV.5183\"; using Pkg; Pkg.clone(pwd()); if VERSION >= v\"1.1.0\"; Pkg.build(\"${JL_PKG}\"; verbose=true); else Pkg.build(\"${JL_PKG}\"); end"', assert: true
```

Comment:
You have two `if`s here with only one `end`.

Reply:
Thanks, same error on L83!